cmd/windup.go

```diff
@@ -165,12 +199,50 @@ type Targets []string
 //
 // AddOptions add options.
 func (r Targets) AddOptions(options *command.Options) (err error) {
+	if len(r) == 0 {
+		err = r.addEmpty(options)
+		return
+	}
```

Comment:
hint: This is where the empty list of targets is detected and the _Emtpy_ ruleset is injected.